src/core/agent/services/multiSigService.ts

```diff
@@ -615,6 +617,46 @@ class MultiSigService extends AgentService {
     };
   }
 
+  async readyMultisigRotation(identifierId: string) {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifierId
+    );
+
+    const multiSig = await this.props.signifyClient
+      .identifiers()
+      .get(metadata.signifyName);
+
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(metadata?.signifyName);
+
+    const nextSequence = (Number(multiSig.state.s) + 1).toString();
```

Comment:
Hmm, we can call `.query` without the sequence number to get the latest. I would prefer this because it's very possible that someone rotates their local (single-sig) member identifier twice instead of once and the sequence numbers will be out of sync.

However, I had to write a bug fix for `.query` on KERIA and the version we have deployed doesn't have this fix. :/

I will create a ticket to change this later, OK for now